crates/compiler/builtins/roc/Dict.roc

```diff
@@ -208,3 +210,413 @@ insertIfVacant = \dict, key, value ->
         dict
     else
         Dict.insert dict key value
+
+# We have decided not to expose the standard roc hashing algorithm.
+# This is to avoid external dependence and the need for versioning.
+# For now, it will just live within the Dict file.
+# If we add non-exposed roc standard library files, this should be moved to it's own file.
+# The current implementation is a form of [Wyhash final3](https://github.com/wangyi-fudan/wyhash/blob/a5995b98ebfa7bd38bfadc0919326d2e7aabb805/wyhash.h).
+# It is 64bit and little endian specific currently.
+LowLevelHasher := { originalSeed : U64, state : U64 } has [
+         Hasher {
+             addBytes,
+             addU8,
+             addU16,
+             addU32,
+             addU64,
+             addU128,
+             addI8,
+             addI16,
+             addI32,
+             addI64,
+             addI128,
+             complete,
+             reset,
+         },
+     ]
```

Comment:
did `roc format` format this like this? If so I think that is a bug of roc format

Reply:
It did pass through `roc format`.

Which part do you think is wrong? This looks pretty normal to me.

Reply:
@ayazhafiz I am not exactly sure how you want this to be formatted, maybe it is a tabbing issue, but If you want this to be formatted differently, can you file an issue for it.